src/api/app/components/card_component.html.haml

```diff
@@ -0,0 +1,15 @@
+.col-12.col-md-6.col-lg-4
```

Comment:
What is the reason for having a fixed width? Having the component adjust to its content would make it more flexible. Here's an example from a PR based on the card component, there's clearly too much whitespace. It could easily have half the width and still be fine.

![whitespace](https://user-images.githubusercontent.com/1102934/218415053-319b8e79-0ad6-400d-ad88-75fc6a666a34.png)

Reply:
Could we not merge this line with the `.card` line below? Having an container for the `.card` container isn't needed.

Reply:
Fixed width is fine I'd say, but not in a hard way. I mean, the card component should not have fixed width, I agree, I will adjust it. But it'd still be good if every usage of the card keeps in mind to set the width for the occurrences of that type, i.e.: all the repository cards in the repository page should have the same width, and all the token cards in the token page should have the same width as well, but the two width for repository and token can be different, that one can.

Anyway, I'm going to adapt here the card component to not set the width, :+1: